cabot/cabotapp/views.py

```diff
@@ -366,7 +367,8 @@ class CheckCreateView(LoginRequiredMixin, CreateView):
     template_name = 'cabotapp/statuscheck_form.html'
 
     def form_valid(self, form):
-        form.instance.created_by = self.request.user
+        if self.request.user is not None and not isinstance(self.request.user, AnonymousUser):
+            form.instance.created_by = self.request.user
```

Comment:
A little hesitant about this since we don't want `AnonymousUser` creating checks on prod...but we have login_required everywhere so should be fine 👍